e2e/kms.go

```diff
@@ -65,6 +65,17 @@ var (
 	}
 )
 
+// encryptionKMSIDList contains kms id of vairous encryption types
+var encryptionKMSIDList = []string{
```

Comment:
consider making the IDs part of the `simpleKMS` struct and put all the KMS instances in a list